docs/source/contributing/tests.rst

```diff
@@ -54,7 +53,7 @@ Running the tests
       
       pytest -v jupyterhub/tests/test_api.py::test_shutdown
 
-   For more information, refer to the `pytest usage documentation <https://pytest.readthedocs.io/en/latest/usage.html>`_.
+   For more details refer `pytest usage documentation <https://pytest.readthedocs.io/en/latest/usage.html>`_
```

Comment:
details is a fine synonym for information, but I'm not sure it's an improvement. The removed comma and `to the` are required for grammar, and should be restored.